deps.bzl

```diff
@@ -80,6 +80,7 @@ def daml_deps():
                 # This should be made configurable in rules_haskell.
                 # Remove this patch once that's available.
                 "@com_github_digital_asset_daml//bazel_tools:haskell-opt.patch",
+                "@com_github_digital_asset_daml//bazel_tools:haskell-cc-wrapper-windows.patch",
```

Comment:
Maybe a comment that this can be removed once https://github.com/tweag/rules_haskell/pull/1464 is merged